### ISSUE TITLE
Error "Variable is not declared (see :help E738)" on function name without g: prefix

### ISSUE THREAD

**Issue by todesking** [Kuniwak/vint#106]
```vim
function! Hoge()
	" Variable is not declared (see :help E738)
	call Hoge()
endfunction
```

**Comment by Kuniwak** [comment]
Thanks, @todesking !

This bug was already fixed on #93, but not released yet.
So the bug will be fixed by the next release [v0.3.1](https://github.com/Kuniwak/vint/milestones/v0.3.1).

**Comment by Kuniwak** [comment]
Now v0.3.1 has been released.
You can upgrade Vint by `pip install -U vim-vint`.
